boards/arm/nucleo_g474re/nucleo_g474re.yaml

```diff
@@ -18,4 +18,5 @@ supported:
   - gpio
   - usb device
   - counter
+  - watchdog
```

Comment:
Move to the end of the list so its (kind of) alpha-sorted.

Reply:
??? There is absolutely no alphabetic sorting in this list , otherwise counter is misplaced, i2c is misplaced, gpio is misplaced ...
But I won't fight for that. done.